src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.PathologyRecordFixtures.generatePathologyRecord;
+
+@ExtendWith(MockitoExtension.class)
+public class BundleMapperTest {
```

Comment:
We've covered the happy path, is there an unhappy path?

Reply:
We're simply building an object here, and at this point all the mandatory data should be available, so I don't think there is an unhappy path here. But this could change as this mapper is extended

Reply:
There are some (logical) branches because of `.ifPresent()`